src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/AbstractTask.kt

```diff
@@ -169,4 +176,79 @@ abstract class AbstractTask : DefaultTask(), AemTask {
         return response
     }
 
+    protected fun deletePackage(installedPackagePath: String, sync: DeploySynchronizer) {
+        val url = sync.htmlTargetUrl + installedPackagePath + "/?cmd=delete"
+
+        logger.info("Deleting package using command: " + url)
+
+        try {
+            val rawHtml = sync.post(url)
+            val response = DeleteResponse(rawHtml)
+
+            when (response.status) {
+                AbstractHtmlResponse.Status.SUCCESS,
+                AbstractHtmlResponse.Status.SUCCESS_WITH_ERRORS -> if (response.errors.isEmpty()) {
```

Comment:
this if in same line looks strange. does it could be wrapped into `{` .. `}` as else branch here? https://kotlinlang.org/docs/reference/control-flow.html